pyramid_swagger/ingest.py

```diff
@@ -62,27 +82,76 @@ def _load_resource_listing(resource_listing):
         )
 
 
-def compile_swagger_schema(schema_dir):
+def generate_resource_listing(schema_dir, listing_base):
+    if 'apis' in listing_base:
```

Comment:
I would not want this to be a valid spec, I think that would be more confusing.  This should  be just a template of  the resource listing, without any `apis`.